### ISSUE TITLE
Browser list has no values,  raising FakeUserAgentError in fake.py", line 140,

### ISSUE THREAD

**Issue by MuhammadRahman-awin** [fake-useragent/fake-useragent#59]
Hi there,

For whatever reason, the self.data['browsers']/self.data_browsers[browser] has only keys, no values, 

>     {u'chrome': [], u'opera': [], u'firefox': [], u'internetexplorer': [], u'safari': []}

Therefore our production cron is broken due to this error

>     raise FakeUserAgentError('Error occurred during getting browser')  # noqa

So I am asking you to return a default browser if choice is empty. 
i.e in line https://github.com/hellysmile/fake-useragent/blob/master/fake_useragent/fake.py#L136
```python
if not self.data_browsers[browser]:
    return random.choice(['firefox', 'chrome', 'opera', 'safari']) # i.e a fallback list
return random.choice(self.data_browsers[browser])
```

Hope you can give a better solution.

**Comment by hellysmile** [comment]
It was fixed yesterday, at version 0.1.8! Can You try to update?

**Comment by MuhammadRahman-awin** [comment]
Oh, didn't notice this and created a PR, going to try now :)
And it works.
